<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>238</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0" colspan="5">
    <widget class="QLineEdit" name="infoShow"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>学历</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QCheckBox" name="banana">
     <property name="text">
      <string>香蕉</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QRadioButton" name="lv2">
     <property name="text">
      <string>中学</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QRadioButton" name="man">
     <property name="text">
      <string>男</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QRadioButton" name="lv4">
     <property name="text">
      <string>研究生</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>性别</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="5">
    <widget class="QLineEdit" name="frult_show"/>
   </item>
   <item row="7" column="0">
    <widget class="QCheckBox" name="apple">
     <property name="text">
      <string>苹果</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QCheckBox" name="all_chose">
     <property name="text">
      <string>全选</string>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QRadioButton" name="lv3">
     <property name="text">
      <string>大学</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QCheckBox" name="pear">
     <property name="text">
      <string>梨</string>
     </property>
    </widget>
   </item>
   <item row="7" column="4">
    <widget class="QCheckBox" name="watermelon">
     <property name="text">
      <string>西瓜</string>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QCheckBox" name="strawberry">
     <property name="text">
      <string>草莓</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>请选择你的喜欢的水果：</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QRadioButton" name="lv1">
     <property name="text">
      <string>小学</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="5">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>————————————————————————————————————————————</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QRadioButton" name="woman">
     <property name="text">
      <string>女</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
